### PR TITLE
docs: Drop unknown abbr milis

### PR DESCRIPTION
**Pull request by tisonkun**
## What's Changed in this PR

This follows up https://github.com/GreptimeTeam/docs/pull/1021/ and reflects https://github.com/GreptimeTeam/greptimedb/pull/4220

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

## Summary by CodeRabbit

- **Documentation**
	- Removed the abbreviation "mils" for milliseconds from the SQL data types reference documents.

### CODE REVIEW

**Comment by tisonkun** [comment]
FYI @etolbakov

**Comment by killme2008** [comment]
Not only this abbreviation. The description must be changed too:

https://github.com/GreptimeTeam/docs/blob/6a8ff5408272955aaa4a191cb071f5c47bd9682b/docs/nightly/en/reference/sql/data-types.md?plain=1#L141

Already supports the casting

**Comment by tisonkun** [comment]
@killme2008 updated.

**Comment by nicecui** [comment]
maybe the content of en and zh should be the same? 🤔

**Comment by tisonkun** [comment]
> maybe the content of en and zh should be the same? 🤔

Yeah. I forgot to update the ZH part ..

File: docs/nightly/en/reference/sql/data-types.md [hunk 1]
```diff
@@ -138,13 +138,19 @@ SELECT now() - INTERVAL '5 minute';
 
 Note that you can also input the interval type using the `INTERVAL 'literal'` format. Using the syntax `'-1 hour -5 minute'::INTERVAL` explicitly casts the string to an interval type, which is how SQL handles type conversion.
 
-GreptimeDB also supports shorthand forms without spaces, such as `3y2mon4h`, but they must be written in the `INTERVAL 'literal'` format:
+GreptimeDB also supports shorthand forms without spaces, such as `3y2mon4h`:
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/data-types.md:141]: please also update here to zh, thank you very much

Comment (tisonkun) [reply docs/nightly/en/reference/sql/data-types.md:141]: Updated.

**Review by nicecui — APPROVED** [review]

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**